packages/client/src/components/FilteringArea/FilteringSection.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const FilteringSection = ({
+  title,
+  data,
+  iconName,
+  checkboxName,
+  selectedValue,
+  setSelectedValue,
+}) => {
+  const iconLink = `/assets/vectors/${iconName}.svg`;
+
+  const handleShowAll = (e) => {
+    const all = document.getElementsByName(checkboxName);
+    for (let i = 0; i < all.length; i += 1) {
+      if (all[i].type === 'checkbox') all[i].checked = false;
```

Comment:
I think this condition could be considered redundant, as it will always be true:
`if (all[i].type === 'checkbox')`
since all of your inputs are assigned `type="checkbox"`.